sway/config/bar.c

```diff
@@ -238,6 +238,8 @@ static void invoke_swaybar(struct bar_config *bar) {
 					bar->swaybar_command ? bar->swaybar_command : "swaybar",
 					"-b", bar->id, NULL};
 			execvp(cmd[0], cmd);
+
+			sway_log_errno(SWAY_ERROR, "Failed to start swaybar");
```

Comment:
Nit: "swaybar" vs "status command", even though they're both "xyz_command" in the config.

Reply:
Hm, I don't understand. This `execvp` is about starting swaybar, not the status command.

Reply:
You add an error log for exec'ing status_command in the {R, and one for exec'ing swaybar_command (this line), but one message says "status command" while the other says "swaybar".